### PR TITLE
Refs 105: symlink containerfile for remote podman building

### PR DESCRIPTION
**Pull request by mshriver**
ripping this from provisioning-backend, we're seeing podman builds fail
and suspect it is the location of the dockerfile.

### CODE REVIEW

**Comment by jlsherrill** [comment]
https://issues.redhat.com/browse/HMSCONTENT-105

File: mk/docker.mk [hunk 1]
```diff
@@ -32,7 +32,9 @@ docker-login:
 
 .PHONY: docker-build
 docker-build:  ## Build image DOCKER_IMAGE from DOCKER_DOCKERFILE using the DOCKER_CONTEXT_DIR
-	$(DOCKER) build $(DOCKER_BUILD_OPTS) -t "$(DOCKER_IMAGE)" -f $(DOCKER_DOCKERFILE) $(DOCKER_CONTEXT_DIR)
+	# spooky action at a distance
+	ln -f $(DOCKER_DOCKERFILE) Containerfile
+	$(DOCKER) build $(DOCKER_BUILD_OPTS) -t "$(DOCKER_IMAGE)" $(DOCKER_CONTEXT_DIR)
```
Comment (jlsherrill) [inline mk/docker.mk:37]: can you add Containerfile to .gitignore?

Comment (jlsherrill) [reply mk/docker.mk:37]: or just rm it after the build?

Comment (mshriver) [reply mk/docker.mk:37]: gitignore'd it, thanks!

**Review by jlsherrill — APPROVED** [review]

**Pull request merged**